web/components/roadmap/MiniRoadmap.tsx

```diff
@@ -0,0 +1,167 @@
+              <img src={`/img/section-header-${sectionName}.svg`} alt="section" height={32} />{" "}
+              <span className="padding-left-105">{publicName}</span>
+            </h2>
+          </AccordionSummary>
+          <AccordionDetails>
+            {roadmap?.steps
+              .filter((step) => step.section === sectionType)
+              .map((step, index, array) => (
+                <div
+                  key={step.step_number}
+                  id={`vertical-content-${step.step_number}`}
+                  className="margin-y-2"
+                >
+                  <div className="fdr fac padding-left-05">
+                    <VerticalStepIndicator
+                      stepNumber={step.step_number}
+                      last={index === array.length - 1}
+                      isOpen={openSteps.includes(step.step_number)}
+                      active={step.step_number === activeStepNumber}
+                      small={true}
+                      completed={isStepCompleted(step, userData)}
+                      key={openSteps.join(",")}
+                    />
+                    <button
+                      className="usa-button--unstyled width-100"
+                      onClick={() => toggleStep(step.step_number)}
+                      aria-expanded={openSteps.includes(step.step_number)}
+                    >
+                      <div className=" step-label sm fdr fjc fac">
+                        <h3
+                          className={`margin-0 font-body-xs line-height-body-2 text-ink ${
+                            step.step_number === activeStepNumber ? "text-primary-dark" : "weight-unset"
+                          }`}
+                          data-step={step.step_number}
+                        >
+                          {step.name}
+                        </h3>
+                        <div className="mla fdc fac">
+                          <Icon className="usa-icon--size-3 text-ink">
+                            {openSteps.includes(step.step_number) ? "expand_less" : "expand_more"}
+                          </Icon>
+                        </div>
+                      </div>
+                    </button>
+                  </div>
+                  <div className="margin-left-5 font-sans-xs">
+                    {openSteps.includes(step.step_number) &&
+                      step.tasks.map((task) => (
+                        <MiniRoadmapTask
+                          key={task.id}
+                          task={task}
+                          active={task.id === props.activeTaskId}
+                          onTaskClick={props.onTaskClick}
+                        />
+                      ))}
+                  </div>
+                </div>
+              ))}
+          </AccordionDetails>
+        </Accordion>
+        <hr className="margin-y-2 bg-base-lighter" />
+      </div>
+    );
+  };
+
+  return (
+    <>
+      {roadmapSections.map((sectionName) => {
```

Comment:
same comment as above - this is shared with RoadmapBySections and could be refactored and shared

Reply:
Addressed.